### PR TITLE
Added Email Authentication

### PR DESCRIPTION
**Pull request by Abh1shekSingh**
## 🛠️ Fixes Issue

Closes: #120 

## 👨‍💻 Changes proposed

1. Added SignUp Email Method
2. Added Login Verification Method
3. Fixed some broken links

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

I have added Signup and login method

## 📷 Screenshots

emil1

### CODE REVIEW

**Comment by Abh1shekSingh** [comment]
@KaizenGirl1111 Please review my PR and merge if possible

**Comment by Abh1shekSingh** [comment]
@KaizenGirl1111 I have also added a way to show Email ID of user at the top of navbar so that user get assured that he is actually logged in

**Comment by Abh1shekSingh** [comment]
@KaizenGirl1111 Shall I add a way that, when User once get Logged in i hide the signup and login pages and show signout instead and once sign out, i revert those changes back ? IF yes i will create one new PR for the Same.

Thank You

**Comment by Abh1shekSingh** [comment]
Check kr lijiye please

File: src/Components/App/Login.jsx [hunk 1]
```diff
@@ -51,7 +67,7 @@ function Login() {
           />
         </div>
 
-        <button type="submit">Login</button>
+        <button onClick={handleLogin} type="submit">Login</button>
```
Comment (KaizenGirl1111) [inline src/Components/App/Login.jsx:70]: good enhancements!

**Review by KaizenGirl1111 — APPROVED** [review]

**Pull request merged**